packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.base.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import {
```

Comment:
Can you move this component to the experiments package?

This will give us an opportunity to evaluate, and to validate readiness for others to consume. It will get published in `@uifabric/experiments` so that you can try it and adjust without worrying about semver.

Readiness means:

There are tests (functional, snapshots, vr tests)
The interfaces are consistent with others
The documentation and examples are clear
It is accessible (keyboarding, high contrast, screen reader aria attributes)
It doesn't have hardcoded colors or strings
Design has signed off (it looks consistent with other things)

We can help to get some of these things going! But lets start in experiments.

Reply:
Got it. Will do first thing tomorrow.